src/ifcblenderexport/blenderbim/bim/module/cobie/operator.py

```diff
@@ -0,0 +1,85 @@
+import bpy
+import blenderbim.bim.module.context.add_context as add_context
+import blenderbim.bim.module.context.remove_context as remove_context
```

Comment:
add_context / remove_context are not needed

Reply:
Sure, by bad 👍